docs/server/source/drivers-clients/websocket-event-stream-api.rst

```diff
@@ -0,0 +1,96 @@
+The WebSocket Event Stream API
+==============================
+
+.. important::
+    This is currently scheduled to be implemented in BigchainDB Server 0.10.
+
+BigchainDB provides real-time event streams over the WebSocket protocol with
+the Event Stream API.
+
+Connecting to an event stream from your application enables a BigchainDB node
+to notify you as events are processed, such as new `validated transactions <#valid-transactions>`_.
+
+
+Demoing the API
+---------------
+
+You may be interested in demoing the Event Stream API with the `WebSocket echo test <http://websocket.org/echo.html>`_
+to familiarize yourself before attempting an integration.
+
+
+Determining Support for the Event Stream API
+--------------------------------------------
+
+In practice, it's a good idea to make sure that the node you're connecting with
+has advertised support for the Event Stream API. To do so, send a HTTP GET
+request to the node's :ref:`Root URL <bigchaindb-root-url>` and check that the
+response contains a ``streams_<version>`` property in ``_links``::
+
+    {
+      "_links": {
+         "streams_v1": "ws://example.com:9984/api/v1/streams/"
```

Comment:
That's implementation specific, isn't it?
My opinion is to allow users to connect to any node in the federation, without specifying an explicit host or a port.
We will go back to the discussion we had yesterday about how do we support overloaded nodes and distribute requests.